Products/PloneKeywordManager/testing.py

```diff
@@ -27,6 +39,9 @@ def setUpPloneSite(self, portal):
         self.applyProfile(portal, 'Products.PloneKeywordManager:default')
         self.applyProfile(portal, 'plone.app.dexterity:default')
 
+        if PLONE_5:
```

Comment:
I believe it does.  I think the point here is to ensure that AT support is available under Plone 5, where it no longer is installed by default.  Without including this, the tests fail, but dexterity tests pass under both Plone 4 and Plone 5 already.